miditools/src/main/java/com/mobileer/miditools/MidiPortSelector.java

```diff
@@ -56,13 +56,13 @@ public MidiPortSelector(MidiManager midiManager, Activity activity,
         mMidiManager = midiManager;
         mActivity = activity;
         mType = type;
-        mAdapter = new ArrayAdapter<MidiPortWrapper>(activity,
+        mSpinner = (Spinner) activity.findViewById(spinnerId);
+        mAdapter = new ArrayAdapter<>(mSpinner.getContext(),
```

Comment:
We lost the <MidiPortWrapper> type. The <> is not supported on JVM < V1.6. I am inclined to think that is OK. What do you think?